addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java

```diff
@@ -48,11 +47,12 @@
     private static Set<String> supportedVoices = getSupportedVoices();
 
     /**
-     * Will support only "MP3" for the moment.
+     * Will support only "MP3" and "OGG" for the moment.
      */
     private static Set<String> getSupportedAudioFormats() {
         Set<String> formats = new HashSet<String>();
-        formats.add(AudioFormat.CODEC_MP3);
+        formats.add("MP3");
```

Comment:
Here, we are defining the API constants: MP3, OGG and later AAC, CAF, ...
These values are relative to VoiceRSS API and not to ESH.
It was of course working because AudioFormat.CODEC_MP3=MP3.

OGG is the value used by the API. For ESH and audio in general, OGG is a container and Vorbis a codec.